servers/src/main/java/tachyon/worker/block/evictor/LRUEvictor.java

```diff
@@ -99,20 +126,12 @@ public EvictionPlan freeSpace(long availableBytes, BlockStoreLocation location)
    */
   @Override
   public void onAccessBlock(long userId, long blockId) {
-    if (mLRUCache.containsKey(blockId)) {
-      mLRUCache.get(blockId);
-    } else {
-      mLRUCache.put(blockId, true);
-    }
+    mLRUCache.put(blockId, true);
```

Comment:
@gpang java.util.LinkedHashSet does exist, but it can only maintain "insertion-order" instead of "access-order" which is needed by LRU.